test/tests_for_sdkmr/ee2_scheduler_test.py

```diff
@@ -1,12 +1,25 @@
 # -*- coding: utf-8 -*-
```

Comment:
These tests are made completely redundant by `Condor_test.py` (below). They're left in the PR to show that they pass (with the changes made, obviously) and to compare the output from the prior version of the test to the current version. In the next PR, they will be deleted.